src/App.js

```diff
@@ -1,8 +1,39 @@
 import React from 'react';
+import PropTypes from 'prop-types';
+
 import './App.scss';
 
 const App = () => (
-  <h1>React sum</h1>
+  <>
+    <h1>React sum</h1>
+    <Sum a={2} b={2} />
+    <Sum a={10} b={0} />
+    <Sum a={-5} b={5} />
+  </>
+);
+
+const Sum = ({
+  a,
+  b,
+}) => (
```

Comment:
No need to extra lines here, code is short enough.
```suggestion
const Sum = ({ a, b }) => (
```